deis-tests-parallel/manifests/deis-tests-parallel-pod.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: deis-tests
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  restartPolicy: Never
+  containers:
+  - name: deis-e2e
+    image: quay.io/deisci/deis-e2e:canary
+    imagePullPolicy: Always
+    command: ["ginkgo"]
+    args: ["-slowSpecThreshold=120.00", "-nodes=30", "-noisyPendings=false", "-progress", "tests/"]
+    env:
+      - name: JUNIT
+        value: "true"
+    volumeMounts:
+    - name: artifact-volume
+      mountPath: /root
+  - name: artifacts
+    image: busybox
+    imagePullPolicy: Always
```

Comment:
need to always pull? seems like since you're just running `tail`, it's not a big deal if you have something outdated. even better would be to specify a specific tag that's not `latest`